File: Android/sukritoberoi_SukritKaurOberoi_2125cse1065_2/GettingStarted/IsANumberPrime.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
  
       // write ur code here
       int n,i,j,c=0;
       int t=scn.nextInt();
       for(i=0;i<t;i++)
       {
           n=scn.nextInt();
           for(j=2;j<n;j++)
           {
               if(n%j==0)
                c++;
           }
           if(c!=0)
            System.out.println("not prime");
           else
            System.out.println("prime");
            c=0;
            
       }
  
   }
  }

File: AR-VR/saumyasinha31_SaumyaSinha_2125cse1144_2/GettingStarted/Is A Number Prime.cpp
#include <iostream>
using namespace std;
int main(int argc, char **argv){
    int t;
    cin >> t;
    int n ;
    while(t--){
        cin>>n;
        int c=0;
        for(int i=2;i<n;i++){
            if(n%i==0){
                c++;
            }
        }
        if(c==0){
            cout<<"prime"<<endl;
        }
        else {
           cout<<"not prime"<<endl;
        }
    }
  return 0;  //write your code here
    
}